UsefulProteomicsDatabases/ProteinDbLoader.cs

```diff
@@ -14,7 +14,7 @@ namespace UsefulProteomicsDatabases
 {
     public static class ProteinDbLoader
     {
-        public static readonly FastaHeaderFieldRegex UniprotAccessionRegex = new FastaHeaderFieldRegex("accession", @"([A-Z0-9_.]+)", 0, 1);
+        public static readonly FastaHeaderFieldRegex UniprotAccessionRegex = new FastaHeaderFieldRegex("accession", @"([A-Z0-9_.-]+)", 0, 1);
         public static readonly FastaHeaderFieldRegex UniprotFullNameRegex = new FastaHeaderFieldRegex("fullName", @"\s(.*?)\sOS=", 0, 1);
         public static readonly FastaHeaderFieldRegex UniprotNameRegex = new FastaHeaderFieldRegex("name", @"\|([^\|][A-Z0-9_]+)", 1, 1);
         public static readonly FastaHeaderFieldRegex UniprotGeneNameRegex = new FastaHeaderFieldRegex("geneName", @"GN=([^ ]+)", 0, 1);
```

Comment:
can you also add the dash here
public static readonly FastaHeaderFieldRegex EnsemblAccessionRegex = new FastaHeaderFieldRegex("accession", @"([A-Z0-9_.-]+)", 0, 1);

Reply:
yes but metamorpheus will not use it. MM uses uniprot regex for everything without the ability to change it. https://github.com/smith-chem-wisc/MetaMorpheus/blob/a8f1201d0ff83040dbc25ae2625ae7f5ed996bb9/TaskLayer/MetaMorpheusTask.cs#L540

Reply:
Okay got it. No worries then. Fine as is